### PR TITLE
Remove multiplexer N^2 behavior

### PR DESCRIPTION
**Pull request by shans**

### CODE REVIEW

File: particles/Multiplexer/Multiplexer.js [hunk 1]
```diff
@@ -78,6 +85,8 @@ recipe
           } catch (e) {
             console.log(e);
           }
+
+          console.log('there');
```
Comment (shaper) [inline particles/Multiplexer/Multiplexer.js]: Debug bits!

**Comment by shans** [comment]
I'm going to add a test to this, I think.

File: particles/Multiplexer/Multiplexer.js (hunk 1 above)
Comment (shans) [reply particles/Multiplexer/Multiplexer.js]: Removed.

**Comment by shans** [comment]
This test is a tad dodgy, y'all. Specifically:

* need to wait (with setTimeout) before providing data, because otherwise the data's already there when the transformer starts up.
* need to wait (with setTimeout) after providing data, because otherwise slots requested by the transformer don't arrive before the end of the test
* *something* isn't resolving properly with the internal recipes so there's a bunch of stuff printed out ; this doesn't change the validity of the test but it's untidy.

I'll keep poking into it.

**Comment by shans** [comment]
Much better!

**Pull request merged**